src/components/common/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+
+import { ReactComponent as Arrow } from 'assets/arrow.svg';
+
+import { MainInput, MainInputWrapper } from './styled';
+
+export default function MainFiltersInput({
+  isOpen,
+  onOpenClick,
+  onCloseClick,
```

Comment:
`on...` уже предполагает, что там обрабатывается действие
```suggestion
  onOpen,
  onClose,
```